dev/tests/integration/testsuite/Magento/MediaStorage/Console/Command/ImageResizeCommandTest.php

```diff
@@ -116,6 +116,7 @@ public function testExecuteWithZeroByteImage()
      *
      * @magentoDataFixture Magento/MediaStorage/_files/database_mode.php
      * @magentoDataFixture Magento/MediaStorage/_files/product_with_missed_image.php
+     * @magentoDbIsolation disabled
```

Comment:
Note for QA: Need to make sure this test is not leaving any data in DB after test run.